src/Symfony/Component/Validator/Constraints/Base64.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Validator\Constraints;
+
+use Symfony\Component\Validator\Constraint;
+
+/**
+ * @author Alexandre Daubois <alex.daubois@gmail.com>
+ */
+#[\Attribute(\Attribute::TARGET_PROPERTY | \Attribute::TARGET_METHOD | \Attribute::IS_REPEATABLE)]
+final class Base64 extends Constraint
+{
+    public const INVALID_STRING_ERROR = '5699d0ba-3cea-4676-9aab-d3303ebd6934';
+
+    protected const ERROR_NAMES = [
+        self::INVALID_STRING_ERROR => 'INVALID_STRING_ERROR',
+    ];
+
+    public bool $allowDataUri = false;
+    public string $message = 'The given string is not a valid Base64 encoded string.';
+
+    public function __construct(array|string $allowDataUri = null, string $message = null, array $groups = null, mixed $payload = null, array $options = null)
```

Comment:
```suggestion
    public function __construct(bool $allowDataUri = false, string $message = null, array $groups = null, mixed $payload = null, array $options = null)
```

Doesn't that work?

Reply:
Of course, my bad 🤦 Thanks for catching this